src/python/grpcio_tests/tests_aio/unit/call_test.py

```diff
@@ -63,20 +63,15 @@ class TestUnaryUnaryCall(AioTestBase):
             self.assertIs(response, response_retry)
 
     async def test_call_rpc_error(self):
-        async with aio.insecure_channel(self._server_target) as channel:
-            empty_call_with_sleep = channel.unary_unary(
-                "/grpc.testing.TestService/EmptyCall",
+        async with aio.insecure_channel("0.1:80") as channel:
+            hi = channel.unary_unary(
+                '/grpc.testing.TestService/UnaryCall',
                 request_serializer=messages_pb2.SimpleRequest.SerializeToString,
                 response_deserializer=messages_pb2.SimpleResponse.FromString,
             )
```

Comment:
optional: switch to using our generated `Stub`.

Reply:
We haven't been using any stub in any of the tests of the test suite, should we start using them?

Or let's use them only once we start giving support/coverage for using the stubs through the Aio module?